### DISCUSSION TITLE
pushover.net & mailbox sensor

### DISCUSSION THREAD

**Discussion by RolfSzimnau** [PricelessToolkit/MailBoxGuard#55] [General]
Has anyone attempted to integrate the Pushover service (https://pushover.net/) with the mailbox sensor project?

I have several cameras, and the Pushover service works great together with this. This led me to wonder if I could get it working with the mailbox sensor as well. I found a useful tutorial on the following website: https://randomnerdtutorials.com/esp32-pushover-notifications-arduino/. I have attempted to follow the instructions and make the necessary corrections, but I’m not very confident in my abilities. Could someone review the attached file for obvious mistakes before I upload it to LoRa?

[Pushover.txt](https://github.com/user-attachments/files/16522596/Pushover.txt)

**Comment by PricelessToolkit** [comment]
Hi Rolf,  for modification use the WhatsApp version, not JSON or MQTT.  But why? What is the advantage compared to HA?

**Reply by RolfSzimnau** [reply]
i don't own HA... somehow i never went down this path... and instead of having whatsapp/callmebot i wanted to consolidate, and pushover works great with a lot of functionality... 

I did modify the whatsapp version, will try to see if my changes works tomorrow when i have more time.

**Comment by RolfSzimnau** [comment]
I've got it working just the way I want. If anyone else is interested, simply update the following variables:

const char* ssid = "InsertSsid";
const char* password = "password";

// Pushover API tokens
const char* apiToken = "";
const char* userToken = "";

Below is the code i used. 
[LoRa_Gateway_Pushover.txt](https://github.com/user-attachments/files/16540257/LoRa_Gateway_Pushover.txt)

![Screenshot_20240808-104406](https://github.com/user-attachments/assets/648912a2-4bb3-4e1e-aad4-ea7f5d5b70e5)
![Screenshot_20240808-104305](https://github.com/user-attachments/assets/9f461018-cdd2-4039-a836-b214f64f1474)
